database/access_rules.go

```diff
@@ -12,7 +12,11 @@
 
 package database
 
-import "time"
+import (
+	"time"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
```

Comment:
seems odd this is coming from the azure provider, not a bit deal, but if its a useful function across providers maybe it should be in a common utils?

Reply:
Ah, it should be from helpers. I'll get that fixed